projects/angular/src/emphasis/_badges.clarity.scss

```diff
@@ -58,80 +40,27 @@
     vertical-align: middle;
     align-items: center;
     justify-content: center;
-    @include mixins.css-var(
-      min-width,
-      clr-badge-min-width,
-      variables.$clr_baselineRem_0_625,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      background-color,
-      clr-badge-background-color,
-      label-variables.getLabelColor(gray, bg-color),
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      height,
-      clr-badge-height,
-      variables.$clr_baselineRem_0_625,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(
-      border-radius,
-      clr-badge-border-radius,
-      variables.$clr_baselineRem_0_416,
-      variables.$clr-use-custom-properties
-    );
-
-    @include mixins.css-var(
-      font-size,
-      clr-badge-font-size,
-      variables.$clr_baselineRem_0_416,
-      variables.$clr-use-custom-properties
-    );
-    @include mixins.css-var(font-weight, clr-badge-font-weight, inherit, variables.$clr-use-custom-properties);
-    @include mixins.css-var(letter-spacing, clr-badge-letter-spacing, inherit, variables.$clr-use-custom-properties);
-    @include mixins.css-var(line-height, clr-badge-line-height, normal, variables.$clr-use-custom-properties);
+    min-width: var(--clr-badge-min-width);
```

Comment:
Move the CSS variable declarations throughout this file to SCSS variables in the _variables file. Import that file here, and use the SCSS variable to set these property declarations.